tests/wdio/home-spec.js

```diff
@@ -0,0 +1,19 @@
+Terra.describeViewports('home', ['tiny', 'huge'], () => {
+  before(() => {
+    const viewport = browser.getViewportSize();
+    viewport.height = 2100;
+    browser.setViewportSize(viewport);
```

Comment:
I would rather we go 'rouge' and use the visual-regression service's `browser.checkDocument([{options}]);` command than manually adjust the viewport to a non-terra defined height

Reply:
https://github.com/zinserjan/wdio-visual-regression-service#usage

Reply:
I'm looking at this and see we are only doing accessibility here. The axe tool by default will scroll to access the whole page so we don't need to resize